user/src/main/java/com/nangman/user/common/feign/HubClient.java

```diff
@@ -0,0 +1,18 @@
+package com.nangman.user.common.feign;
+
+import com.nangman.user.application.dto.HubDto;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.UUID;
+
+@FeignClient(name = "hub")
+public interface HubClient {
+
+    @GetMapping("/hubs/managers/{managerId}")
```

Comment:
Restful 설계를 하면 아래와 같이 요청하는게 더 적절하지 않을까 싶습니다!
```suggestion
    @GetMapping("/hubs?managerId={managerId}")
```

Reply:
API 명세서 바탕으로 작성했는데 논의해보면 좋을거 같습니다!